_test/test_sqw_pageOpMethods/test_binary_ops.m

```diff
@@ -165,20 +167,48 @@ function test_adding_sqw_and_sigvar_with_sqw_1st_operand_returns_sqw(obj)
                 obj.DOUBLE_REL_TOLERANCE);
         end
 
-        function test_adding_sqw_and_sigvar_with_sqw_2nd_operand_returns_sigvar(obj)
-            signal = 1e5 * (1 + rand(size(obj.sqw_obj.data.s)));
-            variance = 1e5 * (1 + rand(size(obj.sqw_obj.data.s)));
+        function test_adding_sqw_and_sigvar_with_sqw_2nd_operand_returns_sqw(obj)
+            signal = (1 + rand(size(obj.sqw_obj.data.s)));
+            variance =(1 + rand(size(obj.sqw_obj.data.s)));
```

Comment:
The previous behaviour of the binary operations sigar+sqw was returning sigvar. This have changed  because sqw contains more information and you can always get sigvar from sqw but the opposite is difficult. As the previous test was returning sigvar, it has been constructed to ignore sqw part (sigvar >> sqw), so within the tested tolderance sqw was ignored.  (sqw has 0-s in places where signar does not). 
As behaviour have changed, I've changed the test to have sqw and sigvar to be of the same order to properly evaluate each contribution. 

I do not see why you would write all this in the test as information about previous implementations is irrelevant in a future.